app/cli/internal/policy/init/init.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policy
```

Comment:
you might be able to simplify a little this package structure, I do not think you need to levels.

I'd just make

```
`internal/policy-devel`
```

and inside you can have an `init.go` file, same package but organized by functionality, would that work?